test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/SkipSection.java

```diff
@@ -153,24 +154,34 @@ public boolean isEmpty() {
         return EMPTY.equals(this);
     }
 
-    static Version[] parseVersionRange(String versionRange) {
-        if (versionRange == null) {
-            return new Version[] { null, null };
+    static List<VersionRange> parseVersionRanges(String rawRanges) {
+        if (rawRanges == null) {
+            return Collections.singletonList(new VersionRange(null, null));
         }
-        if (versionRange.trim().equals("all")) {
-            return new Version[]{VersionUtils.getFirstVersion(), Version.CURRENT};
+        if (rawRanges.trim().equals("all")) {
+            return Collections.singletonList(new VersionRange(VersionUtils.getFirstVersion(), Version.CURRENT));
         }
-        String[] skipVersions = versionRange.split("-");
-        if (skipVersions.length > 2) {
-            throw new IllegalArgumentException("version range malformed: " + versionRange);
+        String[] ranges = rawRanges.split(",");
+        List<VersionRange> versionRanges = new ArrayList<>();
+        for (String rawRange : ranges) {
+            String[] skipVersions = wrapWithSpaces(rawRange).split("-");
```

Comment:
I don't understand why `wrapWithSpaces` is necessary here?

Reply:
Ah, it's to ensure you get two elements in the `skipVersions` array. That's a little subtle.

Reply:
yes, definitely not needed.

Reply:
ok so I see why I put that there. I wanted to allow formats like
`8.0.0 -` where there is no additional space. Also didn't want to make it a special case.
how about using `"8.0.0 -".split("-",-1)` ? -1 there means that the length of array element can have any element. With 0 it means that trailing empty element will not be returned